### PR TITLE
Support two and three channel format.

### PR DESCRIPTION
**Pull request by markweberdev**
Support two and three channel format.

Fix cv2 import.

### CODE REVIEW

File: tracker/iou_tracker.py [hunk 1]
```diff
@@ -19,6 +19,10 @@
 generates video panoptic segmentation with re-ordered identities based on IoU
 overlaps within consecutive frames.
 
+We recommend to use the 3 input channels as it allows for a wide range of
+instance IDs. The evaluator options can be configured with raw_panoptic_format:
+`three_channel_png` to export results in the recommended format.
```
Comment (aquariusjay) [inline tracker/iou_tracker.py:24]: Just a note that please add a link to {kitti_step,motchallenge_step}_test_server_evaluation.md, once those files are ready.
Then users will have a better picture of what is going on.

File: tracker/iou_tracker.py [hunk 2]
```diff
@@ -57,6 +61,9 @@
 flags.DEFINE_string('optical_flow', None,
                     'The path to the optical flow predictions. This folder '
                     'should contain one folder per sequence.')
+flags.DEFINE_integer('input_channels', 2, 'The number of channels from the '
```
Comment (aquariusjay) [inline tracker/iou_tracker.py:64]: Explain in more detail about what you mean by "input_channels" and what does 2 and 3 mean.

File: tracker/iou_tracker.py [hunk 3]
```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
     return list_of_matches, unmatched_instances, unmatched_tracks
 
 
-def read_panoptic_image(path: Text, label_divisor: int) -> np.ndarray:
+def read_panoptic_image_2ch(path: Text, label_divisor: int) -> np.ndarray:
   """Reads in a panoptic image in 2 channel format and returns as np array."""
```
Comment (aquariusjay) [inline tracker/iou_tracker.py:291]: Explain briefly what is "panoptic image in 2 channel"

File: tracker/iou_tracker.py [hunk 4]
```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
...
   with tf.io.gfile.GFile(path, 'rb') as f:
     image = tf.cast(tf.io.decode_image(f.read()), tf.int32).numpy()
+
+  if image.shape[2] == 3 and np.any(image[..., 2] != 0):
```
Comment (aquariusjay) [inline tracker/iou_tracker.py:295]: if image.shape[2] != 2?
more explicit?

File: tracker/iou_tracker.py [hunk 5]
```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
...
+    raise ValueError('The input %s is not in 2 channel format.' % path)
   return image[..., 0] * label_divisor + image[..., 1]
 
 
+def read_panoptic_image_3ch(path: Text, label_divisor: int) -> np.ndarray:
+  """Reads in a panoptic image in 3 channel format and returns as np array."""
```
Comment (aquariusjay) [inline tracker/iou_tracker.py:301]: Briefly explain what is panoptic image in 3 channel

**Review by aquariusjay — COMMENTED** [review]
Thanks, Mark!
A few suggestions.
PTAL.

File: tracker/iou_tracker.py (hunk 1 above)
Comment (markweberdev) [reply tracker/iou_tracker.py:24]: Ack.

File: tracker/iou_tracker.py (hunk 2 above)
Comment (markweberdev) [reply tracker/iou_tracker.py:64]: Done.

File: tracker/iou_tracker.py (hunk 3 above)
Comment (markweberdev) [reply tracker/iou_tracker.py:291]: Done.

File: tracker/iou_tracker.py (hunk 5 above)
Comment (markweberdev) [reply tracker/iou_tracker.py:301]: Done.

File: tracker/iou_tracker.py (hunk 4 above)
Comment (markweberdev) [reply tracker/iou_tracker.py:295]: We still want to support standard RGB images that have only zeros in the third channel. This enables much higher support, as not all tools/libraries support other image formats. Please note, vis.store_raw_predictions will also always store an RGB image even though, the third channel is empty when using 'two_channel_png'.

Comment (aquariusjay) [reply tracker/iou_tracker.py:311]: SG! Thanks for the clarification, Mark!

**Review by aquariusjay — COMMENTED** [review]
LGTM. Thanks, Mark!

**Review by ClaireXie — APPROVED** [review]

**Pull request merged**